### PR TITLE
Tximport Only Job

### PR DESCRIPTION
**Pull request by kurtwheeler**
## Issue Number

#911 

## Purpose/Implementation Notes

This breaks tximport out into it's own job so we can queue it for experiments that are close enough.

## Methods

I used the 25 sample/80% cutoff that was determined by @cansav09 in https://github.com/AlexsLemonade/tximport_partial_run_tests/pull/3, however if any experiment gets to 100% tximport will still be run no matter how few samples it has.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

Unit tests so far, but this needs more.

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kurtwheeler** [comment]
Ok tests are passing, this should be good for review. My game plan for testing is to do it in staging because otherwise I have to run at least 20 salmon samples through. I've found three experiments in staging that I'll queue up a tximport job for:
* experiment with id 256 has 75/88 samples processed -- should work
* experiment with id 22 has 22/23 samples processed -- should not have enough saples
* experiment with id 50 has 39/106 samples processed -- should not have enough percentage

File: workers/nomad-job-specs/tximport.nomad.tpl [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+        LOG_LEVEL = "${{LOG_LEVEL}}"
+      }
+
+      # The resources the job will require.
+      resources {
+        # CPU is in AWS's CPU units.
+        cpu = 1024
+        # Memory is in MB of RAM.
+        memory = 3276 #8
```
Comment (Miserlou) [inline workers/nomad-job-specs/tximport.nomad.tpl]: 8?

Comment (Miserlou) [reply workers/nomad-job-specs/tximport.nomad.tpl]: Also why 3276? Is this enough for large experiments? Should this have an increasing back off?

Comment (kurtwheeler) [reply workers/nomad-job-specs/tximport.nomad.tpl]: I think what happened here is that it was supposed to be 327678. That's already the highest tier for salmon, but it seemed like it's a good starting point for tximport. Based on how these jobs do we can add a increasing back off to it.

Or I can change this right now to start lower and go up to 32678, but I'm thinking tximport will need more RAM than the rest of the salmon pipeline. I'll admit though, I'm not sure how sure I am about that.

What do you think?

Comment (Miserlou) [reply workers/nomad-job-specs/tximport.nomad.tpl]: These are rare enough jobs that we can just start with max RAM. I'd give it more CPU as well.

**Pull request merged**